tests/acceptance/view-tree-test.js

```diff
@@ -186,6 +187,71 @@ test("It should correctly display the view tree", async function(assert) {
   ], 'expected title tips');
 });
 
+test("It should filter the view tree using the search text", async function(assert) {
+  let viewTree = defaultViewTree();
+
+  await visit('/');
+  run(() => {
+    port.trigger('view:viewTree', { tree: viewTree });
+  });
+  await wait();
+
+  let treeNodes = findAll('.js-view-tree-item');
+  assert.equal(treeNodes.length, 3, 'expected some tree nodes');
+
+  fillIn('.js-filter-views input', 'post');
+  treeNodes = findAll('.js-view-tree-item');
+  assert.equal(treeNodes.length, 1, 'expected filtered tree nodes');
+
+  let controllerNames = [];
+  let templateNames = [];
+  let modelNames = [];
+  let viewClassNames = [];
+  let durations = [];
+
+  function textFor(selector, context) {
```

Comment:
Can you move this function outside the tests callbacks so we can reuse the same one to avoid duplication?